Compliling such old versions of GROMACS with modern compilers requires to include <limits>.

@Date:   2024-08-02 
@Author: oliver.stueker@ace-net.ca

diff -ru gromacs-2020.7.orig/src/gromacs/awh/biasparams.cpp gromacs-2020.7/src/gromacs/awh/biasparams.cpp
--- gromacs-2020.7.orig/src/gromacs/awh/biasparams.cpp	2022-02-03 07:40:46.000000000 -0330
+++ gromacs-2020.7/src/gromacs/awh/biasparams.cpp	2024-08-02 13:19:05.393313534 -0230
@@ -49,6 +49,7 @@
 #include <cmath>
 
 #include <algorithm>
+#include <limits>
 
 #include "gromacs/math/functions.h"
 #include "gromacs/mdtypes/awh_params.h"
diff -ru gromacs-2020.7.orig/src/gromacs/modularsimulator/modularsimulator.h gromacs-2020.7/src/gromacs/modularsimulator/modularsimulator.h
--- gromacs-2020.7.orig/src/gromacs/modularsimulator/modularsimulator.h	2022-02-03 07:40:46.000000000 -0330
+++ gromacs-2020.7/src/gromacs/modularsimulator/modularsimulator.h	2024-08-02 13:09:01.067423969 -0230
@@ -41,6 +41,7 @@
 #ifndef GROMACS_MODULARSIMULATOR_MODULARSIMULATOR_H
 #define GROMACS_MODULARSIMULATOR_MODULARSIMULATOR_H
 
+#include <limits>
 #include <queue>
 
 #include "gromacs/mdlib/md_support.h"
